Macros/e3 Includes/e3_Classes_Shaman.inc

```diff
@@ -77,6 +81,22 @@ sub Event_AE_AA(string line)
 	}
 /return
 |----------------------------------------------------------------------------|
+#event Idol "[MQ2] Idol"
+#event Idol "#1# tells you, 'Idol'"
+#event Idol "Soul Idol has expired."
+SUB Event_Idol(line, ChatSender)
+	/if (!${Bool[${Spawn[Soul Idol]}]}) { 
```

Comment:
I think you don't need the "!". This is saying, if there's no soul idol. So it's going to block ever starting the soul idol.

Reply:
This is the only one that really has to be changed for it to work